components/CaseStudyCard.tsx

```diff
@@ -19,7 +19,7 @@ export default function CaseStudyCard({ studies = [] }: ICaseStudyCardProps) {
   return (
     <div className='flex flex-wrap pt-10 lg:grid lg:grid-cols-3 lg:gap-8 lg:text-center'>
       {studies.map((study, index) => (
-        <a key={index} href={`casestudies/${study.id}`}>
+        <a key={index} href={`${study.id}`}>
```

Comment:
Should we create a seperate issue for it?


> Instead of `a`, we should use `Link` tag to navigate internal pages of website.

Should we create a new issue for it?

Reply:
@akshatnema .